Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -339,6 +342,7 @@ namespace AtomSampleViewer
             NewFeaturesSample<SkinnedMeshExampleComponent>("SkinnedMesh"),
             NewFeaturesSample<SsaoExampleComponent>("SSAO"),
             NewFeaturesSample<SSRExampleComponent>("SSR"),
+            NewFeaturesSample<XRRPIExampleComponent>("OpenXR"),
```

Comment:
Recommend making this change. It will grey out the sample if the xrsystem is not registered. Recommend the same for the XRExampleComponent above.
`NewRHISample<XRRPIExampleComponent>("OpenXR", [] { return AZ::RPI::RPISystemInterface::Get()->GetXRSystem()!=nullptr; }),`

Reply:
Okay, I'll get these done and post an update.